### PR TITLE
'line' is a bit optional

### LINKED ISSUE #1151: CSSLinter not working with latest SublimeLinter update
I've updated CSSLinter to latest 1.0.5 and restarted Sublime text a few times after SublimeLinter update, but the error still appears.

```
SublimeLinter: #133 csslint styleold.css ERROR:

 Linter crashed.

Traceback (most recent call last):
  File "lint.backend in C:\Users\Lee\AppData\Roaming\Sublime Text 3\Installed Packages\SublimeLinter.sublime-package", line 97, in execute_lint_task
  File "lint.linter in C:\Users\Lee\AppData\Roaming\Sublime Text 3\Installed Packages\SublimeLinter.sublime-package", line 813, in lint
  File "lint.linter in C:\Users\Lee\AppData\Roaming\Sublime Text 3\Installed Packages\SublimeLinter.sublime-package", line 847, in find_errors
  File "linter in C:\Users\Lee\AppData\Roaming\Sublime Text 3\Installed Packages\SublimeLinter-csslint.sublime-package", line 57, in split_match
TypeError: 'NoneType' object is not iterable

### PR DESCRIPTION
**Pull request by kaste**
Fixes #1151

### CODE REVIEW

**Comment by braver** [comment]
Ok, that makes sense. Want me to try and spend an hour trying to recreate the issue and confirm the fix, or shall we just merge, 🚤 and roll with it? Note that I’m super lazy today 😉

File: lint/linter.py [hunk 1]
```diff
@@ -872,8 +872,10 @@ def split_match(self, match):
         if line:
             match_dict["line"] = int(line) - self.line_col_base[0]
         else:
-            logger.info("'line' is not optional. {}".format(match_dict))
-            return None  # log but do not err for now
+            # `line` is not optional, but if a user implements `split_match`
+            # and calls `super` first, she has still the chance to fill in
```
Comment (FichteFoll) [inline lint/linter.py]: Btw, the gender-neutral pronouns are they/them/their.

File: lint/linter.py [hunk 2]
```diff
@@ -872,8 +872,10 @@ def split_match(self, match):
...
+            # a value on her own.
+            match_dict["line"] = None
```
Comment (FichteFoll) [inline lint/linter.py]: `line = match_dict.set_default("line", None)` in line 871 would make this obsolete.

Comment (kaste) [reply lint/linter.py]: I need to make all falsey values explicit None's

**Pull request merged**